### DISCUSSION TITLE
Checking for chunks of different towns to be in touch

### DISCUSSION THREAD

**Discussion by BrainRTP** [TownyAdvanced/Towny#5812] [Q&A] [answered]
Hi! How can I check that towns are contacting each other, i.e. that their chunks border each other?

**Comment by LlmDl** [comment, marked as answer]
Edit the config's min_distance settings.

**Reply by BrainRTP** [reply]
I forgot to mention that I need to do such a check in another plugin via TownyAPI.

**Reply by LlmDl** [reply]
You're going to have to parse through a Town's TownBlocks using Town#getTownBlocks(). You'll have to check each TownBlock's neighbouring TownBlocks to see if they TownBlock#hasTown() or !TownBlock#isWilderness(), and then TownBlock#getTown().equals(yourtown).
